RadarSDK/RadarAPIClient.m

```diff
@@ -58,7 +58,30 @@ + (NSDictionary *)headersWithPublishableKey:(NSString *)publishableKey {
     };
 }
 
-- (void)getConfig {
++ (void)parseMetaAndConfigure:(NSDictionary *_Nullable)res {
+    if (!res) {
+        return;
+    }
+
+    id metaObj = res[@"meta"];
+    if (metaObj && [metaObj isKindOfClass:[NSDictionary class]]) {
+        NSDictionary *meta = (NSDictionary *)metaObj;
+        id configObj = meta[@"config"];
+        if (configObj && [configObj isKindOfClass:[NSDictionary class]]) {
```

Comment:
I prefer the go-style of `return`-ing early, but this was just an extraction of existing logic. Will probably do that here.